src/woo_publications/publications/api/serializer.py

```diff
@@ -62,12 +62,39 @@ class PublicationSerializer(serializers.ModelSerializer):
         many=True,
         allow_empty=False,
     )
+    publisher = serializers.SlugRelatedField(
+        queryset=Organisation.objects.filter(is_actief=True),
+        slug_field="uuid",
+        help_text=_("The organisation which publishes the publication."),
+        many=False,
+    )
+    verantwoordelijke = serializers.SlugRelatedField(
+        queryset=Organisation.objects.filter(is_actief=True),
+        slug_field="uuid",
+        help_text=_(
+            "The organisation which is liable for the publication and it's contents."
+        ),
+        many=False,
+        allow_null=True,
+        required=False,
+    )
+    opsteller = serializers.SlugRelatedField(
+        queryset=Organisation.objects.filter(is_actief=True),
```

Comment:
From Marco's [comment](https://github.com/GeneriekPublicatiePlatformWoo/registratie-component/issues/66#issuecomment-2459489498) I understood this field should also allow non-active organisations.